src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,44 @@
+import { router, useLocalSearchParams } from 'expo-router';
+import { Image, Text, TouchableOpacity, View, ScrollView } from 'react-native';
+
+import styles from './styles';
+import { uploadCase } from '../../../../../supabase/queries/cases';
+import { Case } from '../../../../../types/types';
+import { formatDate } from '../../utils';
+
+function AddCase() {
+  const { id, title, imageUrl, date, lawFirm, summary } =
+    useLocalSearchParams() as unknown as Case;
+  const addToCases = async () => {
+    uploadCase(id);
+    router.push('/Cases');
+  };
```

Comment:
Line here as well

Reply:
sorry what does this refer to?

Reply:
Like add a empty line between the function and the return statement below